ui/src/app/pages/data/concept/concept-homepage.tsx

```diff
@@ -336,33 +314,36 @@ export const ConceptHomepage = withCurrentWorkspace()(
     }
 
     async searchConcepts() {
-      const {standardConceptsOnly, currentSearchString, conceptsCache, selectedDomain,
-        selectedConceptDomainMap} = this.state;
+      const {standardConceptsOnly, currentSearchString, conceptsCache, selectedDomain, selectedConceptDomainMap, selectedSurvey}
+        = this.state;
       const {namespace, id} = this.props.workspace;
       this.setState({concepts: [], searchLoading: true, searching: true, completedDomainSearches: []});
       const standardConceptFilter = standardConceptsOnly ? StandardConceptFilter.STANDARDCONCEPTS : StandardConceptFilter.ALLCONCEPTS;
       const completedDomainSearches = [];
+      const request = {query: currentSearchString, standardConceptFilter: standardConceptFilter, maxResults: this.MAX_CONCEPT_FETCH};
+      if (!!selectedSurvey) {
+        request['surveyName'] = selectedSurvey;
+      }
+      conceptsApi().domainCounts(namespace, id, request).then(counts => {
+        // Filter Physical Measurements if feature flag disabled
+        const conceptDomainCounts = !environment.enableNewConceptTabs
+          ? counts.domainCounts.filter(dc => dc.domain !== Domain.PHYSICALMEASUREMENT)
+          : counts.domainCounts;
+        this.setState({conceptDomainCounts: conceptDomainCounts});
+      });
       conceptsCache.forEach(async(cacheItem) => {
         selectedConceptDomainMap[cacheItem.domain] = [];
         const activeTabSearch = cacheItem.domain === selectedDomain.domain;
-        const resp = await conceptsApi().searchConcepts(namespace, id, {
-          query: currentSearchString,
-          standardConceptFilter: standardConceptFilter,
-          domain: cacheItem.domain,
-          includeDomainCounts: activeTabSearch,
-          maxResults: this.MAX_CONCEPT_FETCH
-        });
+        if (cacheItem.domain === Domain.SURVEY) {
+          await conceptsApi().searchSurveys(namespace, id, request).then(resp => cacheItem.items = resp);
+        } else {
+          await conceptsApi().searchConcepts(namespace, id, {...request, domain: cacheItem.domain})
```

Comment:
I noticed that the `SearchConceptsRequest` accepted by this call has an optional `surveyName` param. Will that change anything in the response?

Reply:
Response will remain the same. SurveyName is only relevant when clicking a specific survey bucket.